src/main.c

```diff
@@ -353,6 +355,18 @@ int rignumber = 0;
 int rig_comm_error = 0;
 int rig_comm_success = 0;
 
+/*-------------------------------------rotctl-------------------------------*/
+bool rot_control = false;
```

Comment:
set the default values in `init_variables()` in order to allow clean re-initialization.
I know that not all globals are covered, but for new ones this shall be done.